### PR TITLE
Add `--webpack-define` command line option.

### PR DESCRIPTION
**Pull request by davidlehn**

### CODE REVIEW

File: lib/index.js [hunk 1]
```diff
@@ -117,6 +121,9 @@ bedrock.events.on('bedrock-cli.init', () => {
       'Dereference top-level node_module symlinks in watch mode' +
       ' (true, false).',
       /^(true|false)$/i, 'true')
+    .option('--webpack-define <name=value>',
+      'Define process.env.NAME=VALUE. (repeatable)',
```
Comment (gannan08) [inline lib/index.js]: My reading is that the `name` is not prefixed with `process.env.`. Is that true? If that's the case then this comment is misleading.

File: lib/index.js [hunk 2]
```diff
@@ -237,6 +244,10 @@ api.bundle = async (options = {}) => {
     // FIXME: are other modes needed?
     defines['process.env.NODE_ENV'] = '"production"';
   }
+  // use CLI defines
+  for(const [name, value] of command.webpackDefine) {
```
Comment (gannan08) [inline lib/index.js]: My understanding is that this command takes precedence over all other commands for the webpackDefinePlug? Is that correct? If so, we should document this.

File: lib/index.js (hunk 1 above)
Comment (davidlehn) [reply lib/index.js]: Yeah, fixed.  Initial plan was to put all these under `process.env.X`, but that seems to only be feasible for something that always is defined.  And this feature is for arbitrary optional things.  So have to use "process.env.FOO=bar".

File: lib/index.js (hunk 2 above)
Comment (davidlehn) [reply lib/index.js]: Hard to parse that.  The CLI defines will override the couple others above.  That's typical behavior for CLI options.  At some point this should allow config files with arbitrary defines.  Waiting to see what's needed.  I added a few more words to the comment.  The README is spare to begin with, unsure how best to quickly document these things.
